website/templates/project/modal_remove_contributor.mako

```diff
@@ -0,0 +1,122 @@
+<link rel="stylesheet" href='/static/css/pages/remove-contributor-page.css'>
+<div id="removeContributor" class="modal fade">
+    <div data-bind="css: modalSize">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h3 class="modal-title" data-bind="text:pageTitle"></h3>
+            </div>
+            <div class="modal-body" >
+                <div data-bind="if: canRemoveNode() && !pageChanged()">
```

Comment:
Yes, good points. I think it's a good idea to use the comment syntax when possible.